### PR TITLE
helm 2.16.9

### PR DESCRIPTION
**Pull request by JohnGreeley**
helm 2.16.9

### CODE REVIEW

File: Food/helm@2.lua [hunk 1]
```diff
@@ -37,8 +37,8 @@ food = {
         {
             os = "windows",
             arch = "amd64",
-            url = "https://get.helm.sh/helm-v" .. version .. "-windows-amd64.tar.gz",
-            sha256 = "bcec96b343a2cd488c1fb4a6603dc08d282b68bf5de0a360579046cca84728e7",
+            url = "https://get.helm.sh/helm-v" .. version .. "-windows-amd64.zip",
```
Comment (bacongobbler) [inline Food/helm@2.lua:40]: what's the reason behind the change? Helm releases both a tarball and a zipfile for Windows. It's just not published on the release page.

Comment (JohnGreeley) [reply Food/helm@2.lua:40]: The published sha256 for the windows archive on the releases page was for the zip, not the tarball. Made sense to make it match what they have on their github releases.

**Comment by bacongobbler** [comment]
#795
